addon/globalPlugins/remoteClient/session.py

```diff
@@ -1,18 +1,22 @@
 import threading
 import time
-import connection_info
+from . import connection_info
 import gui
 import speech
 import ui
 import tones
 import braille
-import configuration
-import nvda_patcher
+from . import configuration
+from . import nvda_patcher
 from collections import defaultdict
-import connection_info
+from . import connection_info
 import hashlib
 
-class RemoteSession(object):
+EXCLUDED_SPEECH_COMMANDS = (
+	speech.commands.BaseCallbackCommand,

```

Comment:
A drawback of this, is that we now also filter the beep and wave commands. They're still coming in through us monkeypatching tones.beep and nvwave.playWaveFile, though, so I think it's fine for an intermediate solution, as long as we agree on this has to be improved in future.

Reply:
I agree this needs attention, and agree it is outside the scope of this PR.